atlp-rwanda/space-bn-frontend
# #175743408 ft(notification):  implement and integrate notifications

#### What does this PR do?

- Integrate notifications 

#### Description of Task to be completed? 

- Design and implement notification pane
- Set notification context
- Install socket.io-client
- Connect frontend to the backend


#### How should this be manually tested?

- Clone the repo 

- In your home directory  cd this branch **`ft-implement-notifications-pane-#175743408`**

- Then run **`npm install`** to install all dependencies

- Run **`npm start`** to start the server

- In your default browser navigate to **`http://localhost:3000/signin`** to login 

> When successfully reached the dashboard you should see notification icon and click on it to view current notifications

#### Any background context you want to provide?

- To successfully send request to the backend you need to put backend url in `.env` file as **`REACT_APP_BACKEND_URL`**

#### What are the relevant Pivotal Tracker stories?

- [#175743408](https://www.pivotaltracker.com/n/projects/2476677/stories/175743408)
- [#175743417](https://www.pivotaltracker.com/n/projects/2476677/stories/175743417)
- [#175743421](https://www.pivotaltracker.com/n/projects/2476677/stories/175743421)

#### Screenshots (if appropriate)

- N/A

#### Questions:

- N/A
